nvmeadm/src/nvmf_discovery.rs

```diff
@@ -529,5 +529,5 @@ pub fn disconnect(nqn: &str) -> Result<usize, NvmeError> {
             Ok(e)
         })
         .collect();
-    Ok(subsys.unwrap().len())
+    Ok(subsys?.len())
```

Comment:
So it panicked in case of disconnect failures ?

Reply:
Yeah, when it could not find the "/sys/class/nvme/nvmeX/delete_controller"